source/includes/geospatial-places-data-setup.rst

```diff
@@ -0,0 +1,29 @@
+Create a collection ``places`` with the following documents:
+
+.. code-block:: javascript
+
+   db.places.insertMany(
+      [
+         {
+            name: "Central Park",
+            location: { type: "Point", coordinates: [ -73.97, 40.77 ] },
+            category: "Parks"
+         } ,
+         {
+            name: "Sara D. Roosevelt Park",
+            location: { type: "Point", coordinates: [ -73.9928, 40.7193 ] },
+            category: "Parks"
+         } ,
+         {
+            name: "Polo Grounds",
+            location: { type: "Point", coordinates: [ -73.9375, 40.8303 ] },
+            category: "Stadiums"
+         }
+      ])
```

Comment:
Just a formatting nit to bring this more in line with the formatting used on the [insertMany](https://docs.mongodb.com/manual/reference/method/db.collection.insertMany/#examples) page

```suggestion
.. code-block:: javascript

   db.places.insertMany( [
      {
         name: "Central Park",
         location: { type: "Point", coordinates: [ -73.97, 40.77 ] },
         category: "Parks"
      },
      {
         name: "Sara D. Roosevelt Park",
         location: { type: "Point", coordinates: [ -73.9928, 40.7193 ] },
         category: "Parks"
      },
      {
         name: "Polo Grounds",
         location: { type: "Point", coordinates: [ -73.9375, 40.8303 ] },
         category: "Stadiums"
      }
   ] )
```

Reply:
Updated. Backports to follow @jeff-allen-mongo.